helpers/router.js

```diff
@@ -20,18 +20,56 @@ var Router = function () {
 		var router = this;
 
 		Object.keys(config).forEach(function (params) {
+
 			var route = params.split(' ');
 			if (route.length !== 2 || ['post', 'get', 'put'].indexOf(route[0]) === -1) {
 				throw Error('Invalid map config');
 			}
-			router[route[0]](route[1], function (req, res, next) {
-				var reqRelevantInfo = {
-					ip: req.ip,
-					method: req.method,
-					path: req.path
-				};
-				root[config[params]](extend({}, reqRelevantInfo, {'body': route[0] === 'get' ? req.query : req.body}), httpApi.respond.bind(null, res));
-			});
+			if ( 'string' == typeof config[params] || config[params] instanceof String ) {
+				router[route[0]](route[1], function (req, res, next) {
+					var reqRelevantInfo = {
+						ip: req.ip,
+						method: req.method,
+						path: req.path
+					};
+					root[config[params]](extend({}, reqRelevantInfo, {'body': route[0] === 'get' ? req.query : req.body}), httpApi.respond.bind(null, res));
+				});
+			} else if (Array.isArray(config[params])) {
```

Comment:
so can all of this conditional be removed as we are now registering it through `attachMiddlewareForUrls`?